setup.py

```diff
@@ -1,17 +1,27 @@
-from setuptools import setup
+"""
+Packaging for the SDK
+"""
+import setuptools
 
-setup(
+with open("README.md", "r") as f:
+    long_description = f.read()
+
+requirements = ["python-dotenv"]
```

Comment:
Should this be here? It's a dev/test dependency that could be in a `requirements.dev.txt` or similar.